src/js/actions.js

```diff
@@ -146,9 +146,9 @@ export const deleteSubMenu = (appID, menuID) => {
 export const subscribeButton = (appID, buttonName, isSubscribed) => {
     return {
         type: Actions.SUBSCRIBE_BUTTON,
-        appID,
-        buttonName,
-        isSubscribed
+        appID: appID,
+        buttonName: buttonName,
+        isSubscribed: isSubscribed
```

Comment:
the same object key and value

Reply:
@YuriiYefimenko please find it in comment in review:
Suggested change
        appID,
        buttonName,
        isSubscribed
        appID: appID,
        buttonName: buttonName,
        isSubscribed: isSubscribed

reverting this, otherwise it is inconsistent with the rest of the file